### PR TITLE
Refactor ActiveMQ scaler config

### PR DESCRIPTION
**Pull request by wozniakjan**
Refactor activemq scaler config using declarative parser from https://github.com/kedacore/keda/pull/5676

### Checklist

- [x] I have verified that my change is according to the [deprecations & breaking changes policy](https://github.com/kedacore/governance/blob/main/DEPRECATIONS.md)
- [x] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://github.com/kedacore/keda/blob/main/CONTRIBUTING.md#developer-certificate-of-origin-signing-your-work))

Relates to 
https://github.com/kedacore/keda/issues/5797

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -60,6 +60,7 @@ To learn more about active deprecations, we recommend checking [GitHub Discussio
 
 - TODO ([#XXX](https://github.com/kedacore/keda/issues/XXX))
 - **General**: Declarative parsing of scaler config ([#5037](https://github.com/kedacore/keda/issues/5037))
+- *ActiveMQ Scaler*: Refactor ActiveMQ config parser ([#5797](https://github.com/kedacore/keda/issues/5797))
```
Comment (zroubalik) [inline CHANGELOG.md:63]: Please move this to `Other` section, as it is not user facing change. Thanks!

Comment (zroubalik) [reply CHANGELOG.md:63]: Or ideally create one entry in the changelog for all scalers.

Comment (wozniakjan) [reply CHANGELOG.md:63]: > Or ideally create one entry in the changelog for all scalers.

I like that option best, probably no need to pollute changelog with bunch of similar refactors. Given https://github.com/kedacore/keda/pull/5676 already added an entry to changelog, I guess that should be sufficient?

Comment (zroubalik) [reply CHANGELOG.md]: Yeah, the only thing we can do is to also add this https://github.com/kedacore/keda/issues/5797 to that changelog entry, something like this `(#5037] | #5797 )`

**Review by zroubalik — APPROVED** [review]

**Pull request merged**